src/txkube/_memory.py

```diff
@@ -132,7 +137,8 @@ def _delete(self, request, collection, collection_name, name):
     app = Klein()
     @app.handle_errors(NotFound)
     def not_found(self, request, name):
-        request.responseHeaders.setRawHeader(u"content-type", [u"application/json"])
+        # XXX Untested - https://github.com/LeastAuthority/txkube/issues/42
+        request.responseHeaders.setRawHeaders(u"content-type", [u"application/json"])
```

Comment:
Adding a test here as specified in https://github.com/LeastAuthority/txkube/issues/42 would be nice